Test.C

```diff
@@ -11,108 +11,214 @@
 #define NUM_ROUNDS 5
 
 //Define structures
-struct Equipment {
-    char name[NAME_LENGTH]; // name of the piece of Equipment
+struct Equipment
+{
+    char name[NAME_LENGTH];               // name of the piece of Equipment
     char description[DESCRIPTION_LENGTH]; // short description of Equipment
-    int cost; // cost of the Equipment
-    int messageAmount; // number of messages able to be sent
+    int cost;                             // cost of the Equipment
+    int messageAmount;                    // number of messages able to be sent
 };
 
-struct Encryption {
-    char name[NAME_LENGTH]; // name of the encryption method
+struct Encryption
+{
+    char name[NAME_LENGTH];               // name of the encryption method
     char description[DESCRIPTION_LENGTH]; // short description of how encryption works
-    int cost; // cost of the encryption
-    double effectiveness; // how effective the encryption is (percentage)
-
+    int cost;                             // cost of the encryption
+    double effectiveness;                 // how effective the encryption is (percentage)
 };
 
-struct Attack {
-    char name[NAME_LENGTH]; // name of the attack
+struct Attack
+{
+    char name[NAME_LENGTH];               // name of the attack
     char description[DESCRIPTION_LENGTH]; // short description of how the attack works
-    int tier; // level of intensity
+    int tier;                             // level of intensity
 };
 
-struct EncryptionInventory {
-    Encryption encryption; // the data
+struct EncryptionInventory
+{
+    Encryption encryption;     // the data
     EncryptionInventory *next; // the pointer
 };
 
-struct Player {
-    char name[NAME_LENGTH]; // the player's name
+struct Player
+{
+    char name[NAME_LENGTH];                      // the player's name
     int equipmentInventory[NUM_EQUIPMENT] = {0}; // the current equipment the user owns
-    EncryptionInventory encryptionInventory; // the current encryption methods the user owns
-    int currentCredits; // the current number of credits the user has available
+    EncryptionInventory encryptionInventory;     // the current encryption methods the user owns
+    int currentCredits;                          // the current number of credits the user has available
 };
 
-struct Round {
+struct Round
+{
     int attackTier;
     Attack attack;
 };
 
 //Function prototypes
 void displayBriefingMessage(Player *playerPtr);
 
-int main(){
+void equipmentStore()
+{
+    int equipSelect;
```

Comment:
You change how you refer to this variable throughout your code, from `equipSelect` to `equipmentSelect`. I would suggest sticking to `equipmentSelect` since _equip_ is a verb and means something different than _equipment_.
```suggestion
    int equipmentSelect;
```